tests/pytorch/nightly/hf-lm.libsonnet

```diff
@@ -189,7 +189,7 @@ local utils = import 'templates/utils.libsonnet';
         export XLA_USE_BF16=$(XLA_USE_BF16)
       |||,
       tpuVmExtraSetup: |||
-        pip install tensorboardX google-cloud-storage
+        pip install tensorboardX google-cloud-storage pathlib
```

Comment:
Can you also remove the `pathlib` backport here? You shouldn't need it with `python3`

Reply:
Thanks, Will, updated